### PR TITLE
Firefox tv quarantine

### PR DESCRIPTION
**Pull request by Pike**

### CODE REVIEW

File: mozilla-mobile/firefox-tv/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -214,4 +210,45 @@
...
     <string name="hint_press_menu_to_open_overlay_a11y">Press the menu button for %1$s Home, search, settings, and more</string>
 
+    <!-- Firefox Accounts Strings-->
+    <!-- Description for Sent tab buttons (in navigation toolbar & settings, both on the homescreen)
+     that will launch the Firefox Account sign-in experience so users can receive Firefox tabs on their Fire TV.
+     %1$s will be replaced with Firefox. -->
+    <string name="fxa_navigation_item_new">New! Send %1$s tabs to Fire TV</string>
+    <!-- Description for Sent tab button in navigation toolbar
+     that will launch the Firefox Account settings after the user is signed in. -->
+    <string name="fxa_navigation_item_signed_in">Sent tab settings</string>
```
Comment (Pike) [inline mozilla-mobile/firefox-tv/app/src/main/res/values/strings.xml]: I wonder if this is `tabs` or `tab`, i.e., most other strings use plural.

File: mozilla-mobile/firefox-tv/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -214,4 +210,45 @@
...
+    <!-- Instruction text for the Firefox Account sign-in pre-boarding, where we introduce the send tab feature.
+    %1$s will be replaced with Firefox.
+    %2$s will be replaced with the name of our app (i.e. Firefox for Fire TV) -->
+    <string name="fxa_preboarding_instruction">Send a tab from anywhere you use %1$s and see it here. Sign in to %2$s to get started.</string>
+    <!-- Sign-in button text for the Firefox Account sign-in pre-boarding, where we introduce the send tab feature. -->
+    <string name="fxa_preboarding_primary_button">Sign in</string>
+    <!-- Text on the cancel button for the Firefox Account sign-in pre-boarding, where we introduce the send tab feature. -->
+    <string name="fxa_preboarding_secondary_button">Not now</string>
+    <!-- Header for Firefox Accounts onboarding screen, which is shown after signing in. -->
+    <string name="fxa_onboarding_header">Your tabs. Now ready for prime time.</string>
```
Comment (Pike) [inline mozilla-mobile/firefox-tv/app/src/main/res/values/strings.xml]: Is `prime time` a word play on Amazon Prime?

File: mozilla-mobile/firefox-tv/app/src/main/res/values/strings.xml (hunk 1 above)
Comment (dnarcese) [reply mozilla-mobile/firefox-tv/app/src/main/res/values/strings.xml]: Yes, that's a good point. Where do the changes get made? Do I re-land this string in fftv repo?

File: mozilla-mobile/firefox-tv/app/src/main/res/values/strings.xml (hunk 2 above)
Comment (dnarcese) [reply mozilla-mobile/firefox-tv/app/src/main/res/values/strings.xml]: I don't believe that was the intention.

File: mozilla-mobile/firefox-tv/app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Pike) [reply mozilla-mobile/firefox-tv/app/src/main/res/values/strings.xml]: Yes. And with the process we have here with strings in quarantine, you won't need to change the ID. ID changes become necessary once the quarantine PR is merged to master here, and we actually expose the strings to localizers.

File: mozilla-mobile/firefox-tv/app/src/main/res/values/strings.xml (hunk 2 above)
Comment (Pike) [reply mozilla-mobile/firefox-tv/app/src/main/res/values/strings.xml]: OK, I'll leave this up to Delphine on whether a comment here is helping or distracting.

Comment (Delphine) [reply mozilla-mobile/firefox-tv/app/src/main/res/values/strings.xml]: I don't think we need a comment here. Thanks for flagging

File: mozilla-mobile/firefox-tv/app/src/main/res/values/strings.xml (hunk 1 above)
Comment (dnarcese) [reply mozilla-mobile/firefox-tv/app/src/main/res/values/strings.xml]: I don't think this change was brought into this PR, afaict. Can it still be exported before the strings are exposed to localizers so we can keep the id the same?

Comment (Delphine) [reply mozilla-mobile/firefox-tv/app/src/main/res/values/strings.xml]: Sorry, I merged thinking they had all been taken care of already.
We'll need an ID change now :/

**Pull request merged**